### PR TITLE
[NodePrepareFinalize] wrapper of salt cleanup command

### PR DESCRIPTION
**Pull request by mazinamdar**
Signed-off-by: mazinamdar <mazhar.inamdar@seagate.com>

Wrapper around salt cleanup command

### CODE REVIEW

File: lr-cli/cortx_setup/commands/node/prepare/finalize.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+#
+
+from cortx_setup.commands.command import Command
+from cortx_setup.commands import SaltCleanup
+
+
+class NodePrepareFinalize(Command):
+    _args = {
+        'force': {
+            'default': False,
```
Comment (divya-vijay) [inline lr-cli/cortx_setup/commands/node/prepare/finalize.py:25]: Is type not needed here
`'type': bool,`

File: lr-cli/cortx_setup/commands/node/prepare/finalize.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            'optional': True,
+            'action': 'store_true',
+            'help': 'Forcefully apply configuration.'
+        }
+    }
+
+    def run(self, force=False):
```
Comment (divya-vijay) [inline lr-cli/cortx_setup/commands/node/prepare/finalize.py:32]: Should we again pass `=False` in method after making `default: False` ?

**Review by divya-vijay — APPROVED** [review]

File: lr-cli/cortx_setup/commands/node/prepare/finalize.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+from cortx_setup.commands import SaltCleanup
+
+
+class NodePrepareFinalize(Command):
+    _args = {
+        'force': {
+            'default': False,
+            'optional': True,
+            'action': 'store_true',
+            'help': 'Forcefully apply configuration.'
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/node/prepare/finalize.py:28]: Change help message to:
`Over-ride finalize validation checks`

**Review by nitin-seagate — APPROVED** [review]
Minor comment

File: lr-cli/cortx_setup/commands/node/prepare/finalize.py (hunk 2 above)
Comment (mazinamdar) [reply lr-cli/cortx_setup/commands/node/prepare/finalize.py:32]: we need it, when we use it as module we need it

File: lr-cli/cortx_setup/commands/node/prepare/finalize.py (hunk 1 above)
Comment (mazinamdar) [reply lr-cli/cortx_setup/commands/node/prepare/finalize.py:26]: Done

File: lr-cli/cortx_setup/commands/node/prepare/finalize.py (hunk 3 above)
Comment (mazinamdar) [reply lr-cli/cortx_setup/commands/node/prepare/finalize.py]: Done

**Pull request merged**